coremltools/models/deprecation.py

```diff
@@ -0,0 +1,23 @@
+import warnings
+import functools
+
+def deprecated(func=None, str=""):
```

Comment:
You probably want to rename this method to `_deprecated`. It the user should use it directly then you hide it.

Reply:
I have renamed the file as `_deprecation.py`, so that I can still continue using `@deprecated`